frontend/src/Utils/commonFetch.ts

```diff
@@ -59,7 +59,12 @@ export const commonFetch =
     };
 
     try {
-      return validateStatus(await fetch(new Request(`${k8sBasePath}${url}`, { credentials: 'include' }), allOptions));
+      let safeURL = url;
+      if (/^\/\//.test(url)) {
+        // https://github.com/openshift/dynamic-plugin-sdk/pull/55
```

Comment:
Although this is merged, the package it created does funky stuff with the cross dependencies... so we want to fix this and rebuild it.